LoginApplication/app/src/main/java/com/example/webonise/loginapplication/MainActivity.java

```diff
@@ -0,0 +1,185 @@
+import android.support.design.widget.TextInputLayout;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import io.realm.Realm;
+import io.realm.RealmResults;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private final String TAG = getClass().getSimpleName();
+
+    Realm realm;
+    private EditText editEmail;
+    private EditText editPassword;
+
+    private Button btnLogin;
+    private Button btnSignup;
+    private String emailText;
+    private String passwordText;
+
+    private TextInputLayout input_layout_email,input_layout_password;
+    private String email,password;
+    Toast toast;
+
+    SharedPreferences sharedPreferences;
+    public boolean login_status=true;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        sharedPreferences=getSharedPreferences("user",0);
+        Boolean islogin=sharedPreferences.getBoolean("userlogin",false);
+        if (islogin){
+            Intent intentHome = new Intent(this, ActivityMenu.class);
+            startActivity(intentHome);
+            finish();
+            return;
+        }
+        realm = Realm.getDefaultInstance();
+        Log.v(TAG, "In Login onCreate");
+        initViews();
+    }
+
+    private void initViews() {
+
+        editEmail = (EditText) findViewById(R.id.editEmail);
+        editPassword = (EditText) findViewById(R.id.editPassword);
+
+        btnLogin = (Button) findViewById(R.id.btnLogin);
+        btnSignup = (Button) findViewById(R.id.btnSignup);
+
+        input_layout_email=(TextInputLayout)findViewById(R.id.input_layout_loginEmail);
+        input_layout_password=(TextInputLayout)findViewById(R.id.input_layout_loginPassword);
+
+        btnLogin.setOnClickListener(this);
+        btnSignup.setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View v) {
+
+        emailText = editEmail.getText().toString();
+        passwordText = editPassword.getText().toString();
+        switch (v.getId()) {
+            case R.id.btnLogin:
+                onLoginClick();
+                break;
+            case R.id.btnSignup:
+                Intent intentSignup = new Intent(this, SignUpActivity.class);
+                startActivity(intentSignup);
+                break;
+            default:
+                break;
+        }
+    }
+
+    private void onLoginClick() {
+        email=editEmail.getText().toString();
+        password=editPassword.getText().toString();
+        String getEmail;
+        String getPassword;
+        validateUser();
+
+        RealmResults<UserProfile> user = realm.where(UserProfile.class).findAll();
```

Comment:
You can get single result as there will be single user.

UserProfile user = realm.where(UserProfile.class).findFirst();